pistrap

```diff
@@ -114,6 +117,18 @@ find_latest() {
     printf "%s/%s/%s" "${RASPI_MIRROR}" "${RASPI_LATEST_PATH}" "${RASPI_LATEST_IMAGE}"
 }
 
+find_latest_400() {
+    RASPI_LATEST_PATH_400=$(curl -k -L "${RASPI_MIRROR_400}"  2>/dev/null \
+        | sed -n "s/.*href=\"\([^0-9]*[0-9]\{4\}-[0-9]\{2\}-[0-9]\{2\}\).*/\1/p" \
+        | sort -n -t"-" -k2 -k3 -k4 -r \
+        | head -n1
+    )
+    RASPI_LATEST_IMAGE_400=$(curl -k -L "${RASPI_MIRROR_400}/${RASPI_LATEST_PATH_400}/" 2>/dev/null \
+        | sed -n "s/.*href=\"\([^\"]*\.img[\.xz]\{0,3\}\)\".*/\1/p"
+    )
+    printf "%s/%s/%s" "${RASPI_MIRROR_400}" "${RASPI_LATEST_PATH_400}" "${RASPI_LATEST_IMAGE_400}"
+}
```

Comment:
Instead of writing a new function, it would be best to adapt the existing code to use function parameters.